catalog/cataloger_delete_entry.go

```diff
@@ -53,6 +53,7 @@ func (c *cataloger) DeleteEntry(ctx context.Context, repository, branch string,
 		var isCommitted bool
 		err = tx.Get(&isCommitted, sql, args...)
 		committedNotFound := errors.Is(err, db.ErrNotFound)
+		// err is real, and not just a "Not found"
```

Comment:
thought it is clear from the next/prev row as the variable called committedNotFound :)